libs/providers/flagd-web/src/lib/flagd-web-provider.ts

```diff
@@ -0,0 +1,304 @@
+import {
+  EvaluationContext,
+  ResolutionDetails,
+  ErrorCode,
+  StandardResolutionReasons,
+  JsonValue
+} from '@openfeature/js-sdk';
+import {
+  createConnectTransport,
+  createPromiseClient,
+  createCallbackClient,
+  PromiseClient,
+  CallbackClient,
+  ConnectError,
+  Code
+} from "@bufbuild/connect-web";
+import { Struct } from "@bufbuild/protobuf";
+import { Service } from '../proto/ts/schema/v1/schema_connectweb'
+import { Md5 } from 'ts-md5'; // TODO: investigate
```

Comment:
What do you need to investigate?

Reply:
Performance - wondering if sha is a better choice.

Reply:
https://automationrhapsody.com/md5-sha-1-sha-256-sha-512-speed-performance/#:~:text=SHA%2D1%20is%20fastest%20hashing,and%2023.4%25%20for%20longer%20strings.

Thought his is probably HUGELY implementation dependent. Also, sometimes hashing can be offloaded to specialty processors, but I have no idea if chrome is capable of doing this.

Reply:
I'm satisfied with this implementation for the client. We might have to look into web workers or something else for the server.